lib/play_game.py

```diff
@@ -3,27 +3,38 @@
 
 
 class PlayGame:
+
+    current_player = "X"
+    next_player = "O"
+
     def __init__(self, board, ui):
         self.board = board
         self.ui = ui
 
     def start(self):
         self.ui.display("Welcome to TIC TAC TOE!")
         self.board.display_board()
+        self.ui.display(self.current_player + " goes first")
         while not self.board.board_full(self.board.board):
```

Comment:
Is there a way you can lean on the instance that you created instead of passing in the argument here? Doesn't self.board already know the state of self.board.board?

Reply:
I'm not sure if I understand what do you mean here....to change the board_full method to check the self.board.board without accepting an argument?